pytorch_lightning/loops/optimization/optimizer_loop.py

```diff
@@ -478,7 +478,7 @@ def _training_step(self, split_batch: Any, batch_idx: int, opt_idx: int) -> Clos
 
         return result
 
-    def _track_and_norm_grad(self, optimizer: torch.optim.Optimizer) -> Dict[str, float]:
+    def _track_and_norm_grad(self, optimizer: torch.optim.Optimizer, opt_idx: int) -> Dict[str, float]:
```

Comment:
off topic.. but there is a little inconsistency with argument name `opt_idx` here. In the hooks open to the users we have `optimizer_idx` but internally at some places, we have `opt_idx`.